internal/api/v1/application/controller.go

```diff
@@ -1,6 +1,28 @@
 // Application contains the API handlers to manage applications. Except for app environment.
 package application
 
+import (
+	"context"
+
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/namespaces"
+	apierror "github.com/epinio/epinio/pkg/api/core/v1/errors"
+)
+
 // Controller represents all functionality of the API related to applications
 type Controller struct {
 }
+
+func (c Controller) validateNamespace(ctx context.Context, cluster *kubernetes.Cluster, namespace string) apierror.APIErrors {
```

Comment:
Wow, this was repeated a lot. I expected more validations in this func, but since it returns an error, the name is a good fit.

Reply:
Yes, actually I was trying to add some other validations in the controller or utility but it was "too much" for this PR.

I would like also to create only one controller in the Routes, but also that is something completely unrelated atm.